### ISSUE TITLE
Not a usual bug report for  java.lang.reflect.InvocationTargetException

### ISSUE THREAD

**Issue by Pultex** [AppliedEnergistics/Applied-Energistics-2#4246]
other posts like this
#4004
#3777

**Describe the bug**
Server crash for what seem to be unknown reasons

**To Reproduce**
have an AE system it seems connected to some unknown element that makes it not access the API correctly

**Expected behavior**
Not have it crash, 

**Additional context**

http://imgbox.com/g/35pbjILChZ
not sure if you could see a problem in the setup
https://privatebin.net/?64ed47675ad5fb9d#6x9XcKSfHjpuFP7fbgknNa28obaQQXsgZK8R5qHX3E3k

**Environment**
World is part of a modpack
Whole system connected through the P2P system, only 3 direct dense cable link to controller
SP

- Minecraft Version: 1.12.2
- AE2 Version: rv6-stable-7 
- Forge Version:14.23.5.2847

I know there are other posts on this just wanted to make it easier to find for others

things that help for a time

walking not flying helped
staff of the traveler from Ender IO helps [FREEZES the server on AE Chunk unload then runs normal]
Chunk loading the ME core helped  [not with IC2 loader, that did not help]
Breaking the main controller that links the system, replacing it back helps
Removing facades from cables helped
Removing all ME Conduits in the system from Ender IO helped
Removing all exporters going into containers which contains change from one fluid ID to another 
[having the exporter go into a tank then the tank piped into the converting container then into another tank for fluid bus to read from]

**Comment by mindforger** [comment]
viewing those pictures i see an really big AE2 system but missing any ae2 energy cells

can you try adding a bunch and see if it helps? maybe it is a combination of system running out of power and unloading chunks and ae2 addons that cause this

**Comment by Pultex** [comment]
yeah there are only 3 energy cells hidden in the floor to prevent power failure, the system has over 4 billion RF in storage all connected by ID tunnel system.

Thank for looking @mindforger 

the issue is that there is some unknown problem that at some point in time the AE system will crash the server.
Some say micro block touching cab;es cause it oddly enough removing all the facades seems to helped.

aestuff is the only addon for AE that is connected that I know I have in use removing them seem to not effect it as the core chunk is still in load range of the player when it happens.

Like yueh said before to others there must be some mod effecting the system indirectly.

**Comment by Pultex** [comment]
Okay I found that if you have BetterFps mod installed which I did not know it was in the mod pack.
They set it to default to a more efficient math setting, I think it might be making it effect AE.
I changed it to Vanilla Math and now it has not crashed several hours of play testing, moving back an forth though chunks.
This might be one of many things effecting it, though this seems to be one of the largest.
